samples/sensor/veaa_x_3/Kconfig

```diff
@@ -0,0 +1,23 @@
+#
+#  Copyright (c) 2024 Vitrolife A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+mainmenu "VEAA sample application"
+
+config SAMPLE_USE_SHELL
+	bool "Use sensor shell and disable loop"
+	default n
```

Comment:
default n is redundant

Reply:
True. Will fix along with potential `io-channels` changes.